README.rst

```diff
@@ -172,21 +141,11 @@ Note:
   If you have altered the global behavior of ``sbt`` this install may
   not work the way it was intended.
 
-Installing For Developers
-===========================
-
-.. code:: console
-
-   make build
-   pip install -e .
-
-``make build`` will assemble the back-end ``jar`` and move it the ``jars``
-sub-package. The second command will install GeoPySpark in "editable" mode.
-Meaning any changes to the source files will also appear in your system
-installation.
-
 Installing to a Virtual Environment
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: Ask Jake if he's seen issues with this. I had no problem using the standard pip installation
+instructions in a virtualenv. If no problems, perhaps we remove for now.
```

Comment:
I'll say that installing into a virtualenv gave me weird problems when trying to get this thing running in Jupyter.  IPython allows `import geopyspark`, but Jupyter does not.  The `sys.path` is lacking a number of .eggs that are needed (shapely, protobuf, and a couple others).  We'd probably want to figure out if this is a reproducible bug, and if so, provide advice on how to fix it.  Jupyter is expected to be a common use.

Reply:
Gotcha. I just did `pip install jupyter && jupyter notebook` then loaded up each of the demos in the `notebook-demos` folder without issue. I did have to `pip install rasterio` for the "Reading in Sentinel data" demo and `pip install pyproj` for the "Layers" demo, but saw no other errors running each of the demos. None of the demos used protobuf as far as I saw, but a few used shapely, which didn't have any trouble.

This was with gps 0.4.0 installed via `pip` using a "recent" version of virtualenvwrapper installed within a month or so using Python 3.6.4. 

While not explicitly stated anywhere, the way the section is written alludes to this issue only cropping up when GPS is _installed from source_ into a virtualenv. I attempted to test this, but ran into what looks like an unrelated GT compiler error when running `make install` from the current master branch:
```
[info] Compiling 73 Scala sources to /Users/andrew/src/geopyspark/geopyspark-backend/geotrellis/target/scala-2.11/classes...
[error] /Users/andrew/src/geopyspark/geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/SpatialTiledRasterLayer.scala:592: type mismatch;
[error]  found   : Long
[error]  required: Int
[error]         case ps: PartitionStrategy => ps.producePartitioner(math.max(gb.size / 512, 1))
[error]                                                                              ^
[error] /Users/andrew/src/geopyspark/geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/SpatialTiledRasterLayer.scala:636: type mismatch;
[error]  found   : Long
[error]  required: Int
[error]         case ps: PartitionStrategy => ps.producePartitioner(math.max(gb.size / 512, 1))
[error]                                                                              ^
```

So I'll have to investigate that further a bit later.

Reply:
Yeah, interesting.  I'll clarify that I ran into my troubles when installing from source as well.

¯\_(ツ)_/¯

Reply:
I've personally never had issues installing GPS either from source or via pip in a virutalenv.

@jpolchlo That's pretty weird behavior. The only thing I can think of that might cause that is if GPS, Jupyter, and all of the other libraries weren't installed in the same virtualenv.

@CloudNiner Yeah, there is a potential issue when installing into a virtualenv from source when the user does `pip install -e .`. As Python will then look at the path of that directory for GPS, and if that folder is ever moved it'll cause errors.

Reply:
Ok. Is the `pip install -e .` an edge case, or is only useful when developing GPS? I think if the former, we should remove it or fixup the wording to clarify when it would be necessary. If the latter, we can move this section down into the `Developing GeoPySpark -> Installing for Developers` section.

Reply:
I think that `pip install -e .` would only be useful for those developing GPS. I can't see much benefit for someone just looking to use it. That issue isn't specific to GPS either, so I'm not sure if there's really anything we should mention if that's the way they want to set things up.

Reply:
Sounds good. I moved it to a short subsection in the Installation for Developers section. Should be visible, but out of the way.